Django/Meetify/appapi/appapi.py

```diff
@@ -1,63 +1,63 @@
-import os
-import sys
-import spotipy
-from spotipy.oauth2 import SpotifyOAuth
-import mysql.connector
-from pprint import pprint
-
-cnx = mysql.connector.connect(user='admin', password=os.getenv(
-    'MEETIFY_SQL_PASS'), host=os.getenv('MEETIFY_SQL_HOST'), database='meetify')
-cursor = cnx.cursor()
-user_api = spotipy.Spotify(auth_manager=SpotifyOAuth(client_id=os.getenv('SPOTIPY_CLIENT_ID'),
-                                                     client_secret=os.getenv(
-                                                         'SPOTIPY_CLIENT_SECRET'),
-                                                     redirect_uri="http://localhost",
-                                                     scope="user-library-read"))
-userid = user_api.me()['id']
-
-
-def intersect_songs(user_to_compare, current_user=userid):
-    cursor.callproc('intersect_liked_songs', args=(
-        current_user, user_to_compare, '@songs'))
-    song_ids = []
-    for result in cursor.stored_results():
-        for song in result.fetchall():
-            song_ids.append(song[0])
-    return user_api.tracks(song_ids[:3])
-
-
-def update_liked_songs():
-    offset = 0
-    user1_liked_songs = []
-    end = False
-    while (end == False):
-        user1_saved_tracks = user_api.current_user_saved_tracks(
-            limit=50, offset=offset)
-        if (len(user1_saved_tracks['items']) == 0):
-            end = True
-        else:
-            for item in user1_saved_tracks['items']:
-                user1_liked_songs.append(item["track"]["id"])
-            offset = offset + 50
-
-    deleteQuery = (
-        "DELETE FROM user_liked_songs WHERE user_id='" + userid + "';")
-    cursor.execute(deleteQuery)
-    for id in user1_liked_songs:
-        query = ("INSERT INTO user_liked_songs (user_id, song_id) "
-                 "VALUES (%s, %s)")
-        data = (userid, id)
-        cursor.execute(query, data)
-    cnx.commit()
-
-
-def main():
-    update_liked_songs()
-    songs = intersect_songs(sys.argv[1])
-    cursor.close()
-    cnx.close()
-    return songs
-
-
-if __name__ == "__main__":
-    main()
+# import os
```

Comment:
Can we just remove this file if we're commenting everything?

Reply:
> Before I've looked at the code at all, I'm not sure if we can clean up this git history at all. It's a bit messy and reveals our SQL password
> 
> ![image](https://user-images.githubusercontent.com/32147527/106479279-284e3800-6478-11eb-8e8e-6cee5d531723.png)
> 
> Maybe some basic rebases/squashes would be helpful if we don't need the full history. But not a huge deal.

Yes, I can look into cleaning up the history.

As far as the SQL password goes, that was for the old server which is no longer operational, so having that in our history is largely irrelevant.

Reply:
> Can we just remove this file if we're commenting everything?

We could, but I wanted to hang on to this code to use as a reference for future features (mentioned in e579843705b71d1de66df3c36382bca40927883e). Ultimately, it will be removed entirely.

On the flipside, we could just go ahead and remove it in favor of using old commits as a reference.

Thoughts? @segeeslice

Reply:
I think using the old commits is best practice, but I know that's also a bit annoying to deal with. Since we're still starting this out it's not a huge deal either way; I mainly wanted to make sure this was an intentional choice

Reply:
I am also in favor of keeping the old appapi file in the repo for now. I have been using it for reference.